eng/common/scripts/update-docs-ci.ps1

```diff
@@ -5,7 +5,7 @@
 
 # Where possible, this script adds as few changes as possible to the target config. We only 
 # specifically mark a version for Python Preview and Java. This script is intended to be invoked 
-# multiple times. Once for each moniker. Currently only supports "latest" and "preview" artifact selection however.
+# multiple times. Once for each moniker. Currently only supports "latest" and "beta" artifact selection however.
```

Comment:
For eng/common file changes, please follow this workflow:
https://github.com/Azure/azure-sdk-tools/blob/99baaea9b6f0b14b88d042fc5fee96e123280c93/eng/common/README.md#workflow

Reply:
This is referring to something different and shouldn't be updated to beta. This is talking about the docs few which still uses preview as the name.